docs/backups-pitr.md

```diff
@@ -4,8 +4,9 @@ Point-in-time recovery functionality allows users to roll back the cluster to a
 specific date and time. Technically, this feature involves saving operations log
 updates [to the cloud storage](backups-storage.md).
 
-Currently, point-in-time recovery functionality can be used with logical backups
-only.
+Starting from the Operator version 1.15.0, point-in-time recovery functionality
```

Comment:
A bit strange sentence, I would use `Starting from the Operator version 1.15.0, point-in-time recovery functionality can be used with both logical and physical backups.`
But I will leave it up to you, I'm not best with English.